_data/upcoming.yml

```diff
@@ -28,6 +28,15 @@
   twitter: railsclub_ru
   cfp_phrase: CFP closes
   cfp_date: "September 15, 2018"
+  
+- name:  Ruby Summit China 2018
+  location: ZhengZhou, Henan, China
+  dates: "October 13-14, 2018"
+  url: http://rubysummit.cn/
+  twitter: rocLv
+  cfp_phrase: CFP open
```

Comment:
This should be `CFP closes` so that when `cfp_phrase` and `cfp_date` are concatenated, it'll end up reading like this:

> CFP closes September 15, 2018